sdk/python/feast/data_source.py

```diff
@@ -453,77 +306,49 @@ def date_partition_column(self, date_partition_column):
         self._date_partition_column = date_partition_column
 
     @staticmethod
-    def from_proto(data_source):
+    @abstractmethod
+    def from_proto(data_source: DataSourceProto):
         """
         Convert data source config in FeatureTable spec to a DataSource class object.
         """
 
+        if data_source.data_source_class_type:
+            print(f"Data Source class type: {data_source.data_source_class_type}")
```

Comment:
Should we start adopting Python logging instead of print?

Reply:
Oops